### PR TITLE
Consolidate contact UI with PlayerTile and Player page

### PR DESCRIPTION
**Pull request by hammadj**
No longer have two different components to display contact info

### CODE REVIEW

File: packages/web/components/Player/PlayerContacts.tsx [hunk 1]
```diff
@@ -46,6 +48,18 @@ export const PlayerContacts: React.FC<Props> = ({ player }) => {
         }
         return null;
       })}
-    </>
+      {player.ethereum_address ? (
+        <Button
+          as="a"
+          href={`https://etherscan.com/address/${player.ethereum_address}`}
+          target="_blank"
```
Comment (dan13ram) [inline packages/web/components/Player/PlayerContacts.tsx:55]: does this also set `rel="noreferrer noopener"`?
if not we need to set it.

**Review by dan13ram — DISMISSED** [review]

File: packages/web/components/Player/PlayerContacts.tsx (hunk 1 above)
Comment (pakokrew) [reply packages/web/components/Player/PlayerContacts.tsx:55]: If it does not, could be great to create a property `external` on `Button` that sets both `target="_blank"` and `rel="noreferrer noopener"`

**Review by pakokrew — APPROVED** [review]
MyMeta start looking neat !

File: packages/web/components/Player/PlayerContacts.tsx (hunk 1 above)
Comment (hammadj) [reply packages/web/components/Player/PlayerContacts.tsx:55]: Yea, the ds package needs a big refactor to use all the new chakra features that make this sort of thing much easier: https://chakra-ui.com/docs/features/chakra-factory

Added it manually for now

**Pull request merged**